megamek/src/megamek/common/MechSummaryCache.java

```diff
@@ -489,8 +489,8 @@ private boolean loadMechsFromDirectory(Vector<MechSummary> vMechs,
             Set<String> sKnownFiles, long lLastCheck, File fDir,
             boolean ignoreUnofficial) {
         boolean bNeedsUpdate = false;
-        loadReport.append("  Looking in ").append(fDir.getPath())
-                .append("...\n");
+//        loadReport.append("  Looking in ").append(fDir.getPath())
```

Comment:
Should probably uncomment these guys?

Reply:
Right :) Although these clutter up the log.